src/DependencyInjection/Providers/TwitchHelixProviderConfigurator.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * OAuth2 Client Bundle
+ * Copyright (c) KnpUniversity <http://knpuniversity.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @Author Erdem Uyanik
+ */
+
+namespace KnpU\OAuth2ClientBundle\DependencyInjection\Providers;
+
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+class TwitchHelixProviderConfigurator implements ProviderConfiguratorInterface
+{
+    public function buildConfiguration(NodeBuilder $node)
+    {
+        // no custom options
+    }
+
+    public function getProviderClass(array $config)
+    {
+        return 'Vertisan\OAuth2\Client\Provider\TwitchHelix';
```

Comment:
Can you add `use` statement for this and below in `getClientClass()`? Then use `TwitchHelix::class`.

This is not done this way in most providers, simply because, iirc, they predate PHP 5.5 when this was added :)